### PR TITLE
Update to Student Home Page Visuals

### PR DESCRIPTION
**Pull request by LoganDemaray316**
Change to files:
EventItem.vue
RecentItem.vue
RepertoireItem.vue
StudentHomePage.vue

Changes visuals to be placed correctly with an updated design.

### CODE REVIEW

File: src/views/StudentHomeDashboard.vue [hunk 1]
```diff
@@ -2,56 +2,62 @@
...
               </v-row>
-              <v-row><RepertoireItem /></v-row>
+              <RepertoireItem />
             </v-card>
           </v-col>
           <v-col cols="5">
-            <v-card elevation="12">
-              <v-card-title>Quicklinks</v-card-title>
-              <v-btn>Make this button pretty</v-btn>
+            <v-card elevation="12" class="fill-height">
```
Comment (ethanimooney) [inline src/views/StudentHomeDashboard.vue:22]: Make sure all the main cards have the mainBlur class

File: src/components/EventsItem.vue [hunk 2]
```diff
@@ -1,27 +1,57 @@
...
-        <v-row>
-          <v-avatar><img src="../assets/Unknown.jpeg" /></v-avatar>
-          Person Name
-        </v-row>
-      </v-card>
-      Music Selction
-      <v-row>
-        <v-avatar><img src="../assets/Unknown.jpeg" /></v-avatar>
-        Person Name
+    <v-card color="blue">
```
Comment (ethanimooney) [inline src/components/EventsItem.vue:3]: This isn't even the right blue -- you can copy the gradient code from figma and make a CSS class for it to assign here.

File: src/views/StudentHomeDashboard.vue [hunk 3]
```diff
@@ -2,56 +2,62 @@
...
+        <v-row class="fill-height">
           <v-col cols="7">
-            <v-card elevation="12">
-              <v-row align="center" justify="center" class="text-center">
-                <v-col cols="8">
-                  <v-card-title>Repertoire</v-card-title>
-                </v-col>
-                <v-col cols="4">
-                  <v-btn elevation="2">View All</v-btn>
+            <v-card class="fill-height mainBlur">
```
Comment (ethanimooney) [inline src/views/StudentHomeDashboard.vue:7]: Lets make sure all the main cards have a rounded class in them - rounded-lg should do the right size.

File: src/components/RepertoireItem.vue [hunk 4]
```diff
@@ -1,15 +1,21 @@
 <template>
   <v-col cols="12">
-    <v-card elevation="20">
-      <v-row align="center" justify="center" class="text-center">
-        <v-col cols="2">
-          <v-avatar><img src="../assets/Unknown.jpeg" /></v-avatar>
+    <v-card>
```
Comment (ethanimooney) [inline src/components/RepertoireItem.vue:3]: You should be able to make a CSS class for this background too.

File: src/views/StudentHomeDashboard.vue [hunk 5]
```diff
@@ -2,56 +2,62 @@
...
+              <v-row>
+                <v-col>
+                  <v-card-title class="font-weight-bold darkGray--text pt-0">
+                    <v-col>Repertoire</v-col>
+                    <v-col class="text-right">
+                      <v-btn rounded="pill" elevation="2">View All</v-btn>
```
Comment (ethanimooney) [inline src/views/StudentHomeDashboard.vue:13]: Add a second button to the left of this titled "Add new" - I just added that

File: src/views/StudentHomeDashboard.vue [hunk 6]
```diff
@@ -2,56 +2,62 @@
...
-            <v-col cols="8">
-              <v-card-title>Your Events</v-card-title>
-            </v-col>
-            <v-col cols="4">
-              <v-btn elevation="2">View All</v-btn>
+        <v-card elevation="12" class="rounded-card fill-height">
+          <v-card-title class="font-weight-bold darkGray--text py-2">
+            <v-col>Your Events</v-col>
+            <v-col class="text-right">
+              <v-btn rounded="pill" elevation="2">View All</v-btn>
```
Comment (ethanimooney) [inline src/views/StudentHomeDashboard.vue:57]: This should be a drop down that can toggle between Upcoming and Past

File: src/components/EventsItem.vue [hunk 7]
```diff
@@ -1,27 +1,57 @@
...
+      <v-col cols="12">
+        <v-card>
+          <v-card-title class="font-weight-bold darkGray--text py-1 px-0">
+            <v-col>Recital Hearing</v-col>
+            <v-col class="text-right">
+              <v-btn rounded="pill" elevation="2">View All</v-btn>
```
Comment (ethanimooney) [inline src/components/EventsItem.vue:9]: These buttons should say Edit

**Review by ethanimooney — CHANGES_REQUESTED** [review]
I have more edits to add but this should be enough for now

File: src/views/StudentHomeDashboard.vue (hunk 1 above)
Comment (LoganDemaray316) [reply src/views/StudentHomeDashboard.vue:22]: Fixed

File: src/components/EventsItem.vue (hunk 2 above)
Comment (LoganDemaray316) [reply src/components/EventsItem.vue:3]: That was easy

File: src/views/StudentHomeDashboard.vue (hunk 3 above)
Comment (LoganDemaray316) [reply src/views/StudentHomeDashboard.vue:7]: Done

File: src/components/RepertoireItem.vue (hunk 4 above)
Comment (LoganDemaray316) [reply src/components/RepertoireItem.vue:3]: 👍

File: src/views/StudentHomeDashboard.vue (hunk 5 above)
Comment (LoganDemaray316) [reply src/views/StudentHomeDashboard.vue:13]: I guess

File: src/components/EventsItem.vue (hunk 7 above)
Comment (LoganDemaray316) [reply src/components/EventsItem.vue:9]: Coolio

File: src/views/StudentHomeDashboard.vue (hunk 6 above)
Comment (LoganDemaray316) [reply src/views/StudentHomeDashboard.vue]: Done

**Review by ethanimooney — APPROVED** [review]
LGTM, thx

**Pull request merged**